modules/@apostrophecms/page/ui/apos/logic/AposPagesManager.js

```diff
@@ -118,7 +118,8 @@ export default {
       return this.moduleOptions.canCreate;
     },
     checkedTypes() {
-      return this.checkedDocs.map(doc => doc.type);
+      const types = this.checkedDocs.map(doc => doc.type);
+      return [ ...new Set(types) ];
```

Comment:
This is responsability of the one building and passing `checkedTypes` to makes sure types are uniques in the list.